frontend/packages/dev-console/src/components/formik-fields/ResourceLimitField.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import cx from 'classnames';
+import { useField, useFormikContext, FormikValues } from 'formik';
+import { FormGroup, ControlLabel, HelpBlock } from 'patternfly-react';
+import { RequestSizeInput } from '@console/internal/components/utils';
+import { getValidationState } from './field-utils';
+import { ResourceLimitFieldProps } from './field-types';
+
+const ResourceLimitField: React.FC<ResourceLimitFieldProps> = ({
+  inputLabel,
+  name,
+  unitName,
+  helpText,
+  ...props
+}) => {
+  const [field, { touched, error }] = useField(name);
+  const { setFieldValue, setFieldTouched } = useFormikContext<FormikValues>();
+
+  const onRequestSizeInputChange = (val) => {
+    setFieldValue(name, _.toNumber(val.value));
+    setFieldValue(unitName, val.unit);
+    setFieldTouched(name, true);
+  };
+
+  return (
+    <FormGroup controlId={`${name}-field`} validationState={getValidationState(error, touched)}>
+      <ControlLabel className={cx({ 'co-required': props.required })}>{inputLabel}</ControlLabel>
+      <RequestSizeInput
+        placeholder={inputLabel}
```

Comment:
We typically don't repeat the label as placeholder text. I would remove placeholder in this case.

Reply:
Remove this placeholder.